File: DSA/Vikash_kumar/Day_2_Question_1.py
#Day2-Question1-
threshold = int(input("Enter the threshold number of students : "))
arrival_times = []
arrival_times = input("Enter the arrival times with the space : ").split(" ")
for i in range(len(arrival_times)):
    arrival_times[i] = int(arrival_times[i])

count = 0
for i in arrival_times:
    if i > 0:
      count += 1
    elif i == 0:
      pass
    else:
      count -= 1
if count >= 0:
  print("YES") 
else:
  print("NO")         

File: DSA/Arshdeep-Kaur/Day2_Question1.cpp
#include <iostream>
using namespace std;

int main() {
    int arrival_times[100], n, threshold;
    cout << "Enter the threshold number of students: ";
    cin >> threshold;
    cout << "Enter the number of arrival times: ";
    cin >> n;
    cout << "Enter the arrival times: ";
    for (int i=0; i<n; i++) {
        cin >> arrival_times[i];
    }
    int count = 0;
    for (int i=0; i<n; i++) {
        if (arrival_times[i] < 0) {
            count++;
        }
    }

    if (count < threshold) {
        cout << "YES" << endl;
    }
    else {
        cout << "NO" << endl;
    }
}
